### ISSUE TITLE
Weird resizeGrid behaviour

### ISSUE THREAD

**Issue by ThiagoFacchini** [bokuweb/react-rnd#161]
Hello Guys,

Really appreciate this wrapper around 2 of the most usable libraries for UI, although I'm struggling with a very weird behaviour regarding resizeGrid.

I'm using Chrome 60.0 for Mac.

Below my code:
`<Rnd
	default={{
		x: 0,
		y: 0,
		width: this.state.canvasWidth,
		height: this.state.canvasHeight
	}}
	bounds='parent'
	className={styles.canvasRenderer}
	resizeGrid={[128, 128]}
	onResizeStop={ (event, direction, refToElement, delta) => {
		console.log(delta)
		this._updateCanvasRendererSize(delta)
	}}
>
	<canvas
		id={this.props.canvasId}
		width={this.state.canvasWidth}
		height={this.state.canvasHeight}
		className={classNames(styles.canvasObj)}
	/>
</Rnd>`

Basically on the very first resizing it does resize but it doesn't respect what I defined on the resizeGrid property which is 128 / 128. After that, it then works as expected. I got absolutely no clue on what can be possibly happening therefore would appreciate any suggestion.

Here's the video that shows the issue (thought helpful )
https://www.youtube.com/watch?v=9NK33nbdSSY

Thanks in advance,
TF

**Comment by bokuweb** [comment]
@ThiagoFacchini Could you please provide reproduced project on webpackbin with latest rnd? https://www.webpackbin.com/bins/-Ku4nRhImIfnt9N08lGu 

thanks

**Comment by bokuweb** [comment]
@ThiagoFacchini Could you please provide reproduced project in codesandbox?

https://codesandbox.io/s/y3997qply9
